internal/saucecloud/cloud.go

```diff
@@ -200,6 +208,26 @@ func (r *CloudRunner) runJob(opts job.StartOptions) (j job.Job, interrupted bool
 	return j, false, nil
 }
 
+func (r *CloudRunner) dryRunForMobile(opts job.StartOptions) error {
```

Comment:
The content in here has nothing to do with mobile ⚠️  In fact it's (almost) the opposite, since only browser (including mobile browsers) tests have a project archive. Native mobile tests don't have a project archive, so running this code block for espresso or xcuit (i.e. the **actual** mobile frameworks) is not even correct